From 4e16c66408df1e3e911127fab2d2944cbbe124cb Mon Sep 17 00:00:00 2001
From: "hui.an" <hui.an@amlogic.com>
Date: Mon, 9 Jun 2014 11:21:03 +0800
Subject: [PATCH 4310/5965] PD #90803: fixed video display abnormal

fixed that the video display is abnormal for the little window
when playing by the 3st apk,such as shouhu apk.
---
 drivers/amlogic/amports/vpp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/amlogic/amports/vpp.c b/drivers/amlogic/amports/vpp.c
index 44521f7c07f5..7276840fb3f5 100755
--- a/drivers/amlogic/amports/vpp.c
+++ b/drivers/amlogic/amports/vpp.c
@@ -314,9 +314,9 @@ vpp_process_speed_check(s32 width_in,
     }
 
     return SPEED_CHECK_DONE;
-
 #else
-    if (video_speed_check_width * video_speed_check_height * height_out > height_screen * width_in * height_in) {
+    /* if (video_speed_check_width * video_speed_check_height * height_out > height_screen * width_in * height_in) { */
+    if (height_out > height_in) {
         return SPEED_CHECK_DONE;
     }
 
-- 
2.19.0

